tidy3d/plugins/dispersion/fit_fast.py

```diff
@@ -698,147 +91,26 @@ def fit(
             Best fitting result: (dispersive medium, weighted RMS error).
         """
 
-        if max_num_poles < min_num_poles:
-            raise ValidationError(
-                "Dispersion fitter cannot have 'max_num_poles' less than 'min_num_poles'."
-            )
-
-        omega = PoleResidue.angular_freq_to_eV(PoleResidue.Hz_to_angular_freq(self.freqs[::-1]))
-        eps = self.eps_data[::-1]
-
-        init_model = FastFitterData.initialize(
-            omega, eps, eps_inf, advanced_param or AdvancedFastFitterParam()
+        omega_data = PoleResidue.Hz_to_angular_freq(self.freqs[::-1])
+        eps_data = self.eps_data[::-1]
```

Comment:
Unrelated question to this PR: does fitter only work with descending frequency list?

Reply:
Good call, it should work regardless of the order. Let me remove this `[::-1]`. I think this was an artifact from early testing and debugging of the fitter.